variables.md

```diff
@@ -1,204 +1,195 @@
 # VARIABLES
 
-A variable is a container for a data which provides a way of labeling data with a descriptive name. Variables label and store data in memory,update information stored in it and Reference or “get” information stored in it.
-
-It is important to distinguish that variables are not values; they contain values and represent them with a name.
-
+A variable is a container for storing data. It is used to label data with a descriptive name. Variables are not values instead they contain values and represent them with a name.
```

Comment:
I am not very sure about this definition. Some variables, depending on the data type can only store the value location and not the value itself. Can you investigate this? I may be wrong in this context.

Reply:
Are you referring to a scenario like below?:
```js
const name = "jamiu";
const firstname = name;
console.log(firstname);//outputs jamiu
```